app/scripts/modules/core/src/managed/managed.dataSource.ts

```diff
@@ -34,6 +34,14 @@ module(MANAGED_RESOURCES_DATA_SOURCE, []).run([
       application.securityGroups.ready().then(() => addManagedResourceMetadataToSecurityGroups(application), noop);
     };
 
+    const loadEnvironments = (application: Application) => {
+      return ManagedReader.getEnvironmentsSummary(application.name);
+    };
+
+    const addEnvironments = (_application: Application, data: IManagedApplicationSummary) => {
```

Comment:
Should this be `IManagedApplicationEnvironmentsSummary`? (or `IManagedApplicationSummary<'resources' | 'artifacts' | 'environments'>` if you do that thing I commented about)